### PR TITLE
changed-readme-for-python-interpreter-for-apache-zeppelin

### PR DESCRIPTION
**Pull request by d3adp007**
### What is this PR for?
Fixed some of the grammatical issues with the documentation for Python Interpreter for Apache Zeppelin.

### What type of PR is it?
Documentation 

### What is the Jira issue?
* Issue link :- https://issues.apache.org/jira/browse/ZEPPELIN-4111

### CODE REVIEW

**Comment by Leemoonsoo** [comment]
Thanks for the fix! LGTM

File: python/README.md [hunk 1]
```diff
@@ -38,13 +38,13 @@ mvn -Dpython.test.exclude='' test -pl python -am
...
+ * [Py4J](https://www.py4j.org/) Python and Java libraries are used to load input zeppelin Java class into the python process (make java code with python code !). Therefore the interpreter can directly create Zeppelin input form inside the Python process (and eventually with some python variable already defined). JVM opens a random open port to be accessible from python process.
 
- * JavaBuilder can't send SIGINT signal to interrupt paragraph execution. Therefore interpreter directly  send a `kill SIGINT PID` to python process to interrupt execution. Python process catch SIGINT signal with some code defined in bootstrap.py
+ * JavaBuilder can't send SIGINT signal to interrupt paragraph execution. Therefore interpreter will directly  send a `kill SIGINT PID` to python process to interrupt execution. Python process catches SIGINT signal with some code defined in bootstrap.py
```
Comment (felixcheung) [inline python/README.md]: extra space in `directly  send`

File: python/README.md [hunk 2]
```diff
@@ -38,13 +38,13 @@ mvn -Dpython.test.exclude='' test -pl python -am
...
 
  * Matplotlib figures are displayed inline with the notebook automatically using a built-in backend for zeppelin in conjunction with a post-execute hook.
 
- * `%python.sql` support for Pandas DataFrames is optional and provided using https://github.com/yhat/pandasql if user have one installed
+ * `%python.sql` support for Pandas DataFrames is optional but can be downloaded from [here](https://github.com/yhat/pandasql) if user has not one installed.
```
Comment (felixcheung) [inline python/README.md]: `if user has not one installed.` -> `if user does not have one installed.`

Comment (d3adp007) [reply python/README.md]: Sure thing!

**Comment by felixcheung** [comment]
LGTM!

**Comment by zjffdu** [comment]
Thanks @d3adp007 ， merging

**Pull request closed without merging**